api/src/main/java/com/okta/sdk/client/ClientBuilder.java

```diff
@@ -206,6 +208,11 @@
     String DEFAULT_CLIENT_PROXY_HOST_PROPERTY_NAME = "okta.client.proxy.host";
     String DEFAULT_CLIENT_PROXY_USERNAME_PROPERTY_NAME = "okta.client.proxy.username";
     String DEFAULT_CLIENT_PROXY_PASSWORD_PROPERTY_NAME = "okta.client.proxy.password";
+    String DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME = "okta.client.authorizationMode";
+    String DEFAULT_CLIENT_CLIENT_ID_PROPERTY_NAME = "okta.client.clientId";
+    String DEFAULT_CLIENT_SCOPES_PROPERTY_NAME = "okta.client.scopes";
+    String DEFAULT_CLIENT_PRIVATE_KEY_KEY_FILE_PATH_PROPERTY_NAME = "okta.client.privateKey.keyFilePath";
```

Comment:
`okta.client.privateKey` should be the file path, which makes this property name incorrect.  It should be 

```
String DEFAULT_CLIENT_PRIVATE_KEY_PROPERTY_NAME = "okta.client.privateKey"
```

Reply:
Noted.